CONTRIBUTING.md

```diff
@@ -1,4 +1,68 @@
-
+# The Build
```

Comment:
Where are instructions for how to developer locally? Does that go in read me?

Reply:
Scroll down to `Development Workflows`